test/metabase/models/field_values_test.clj

```diff
@@ -307,11 +307,13 @@
       (doseq [[id update-map] [[sandbox-id {:field_id 1}]
                                [sandbox-id {:type :full}]
                                [sandbox-id {:type nil}]
+                               ;; this one should be ok, but toucan doesn't give the hook enough info to know better
```

Comment:
Why should it be ok? `:type` is non-nullable in the database so this error is redundant, but it still seems like the correct behaviour to me

Reply:
The database has "full" as the default value, so this would be a no-op.

Reply:
Also the hook inserts an explicit "full" for redundancy / avoiding dependence on storage detail.